src/coordinator/ts/m3db/types.go

```diff
@@ -88,19 +88,24 @@ type ConsolidatedSeriesBlock struct {
 	consolidationFunc    ConsolidationFunc // nolint
 }
 
-func (c ConsolidatedSeriesBlock) beyondBounds(multiSeriesBlock MultiSeriesBlock) bool {
-	if c.Metadata.Bounds.Start != multiSeriesBlock.Metadata.Bounds.Start || c.Metadata.Bounds.End != multiSeriesBlock.Metadata.Bounds.End {
+// func (c ConsolidatedSeriesBlock) equalBounds(multiSeriesBlock MultiSeriesBlock) bool {
+// 	if c.Metadata.Bounds.Start != multiSeriesBlock.Metadata.Bounds.Start || c.Metadata.Bounds.End != multiSeriesBlock.Metadata.Bounds.End {
+// 		return false
+// 	}
+// 	return true
+// }
+
+func equalBounds(boundOne, boundTwo block.Bounds) bool {
+	if boundOne.Start != boundTwo.Start || boundOne.End != boundTwo.End {
```

Comment:
Use `.Equal()` for time comparisons rather than `==`, otherwise it does some weird comparisons across Location and clock settings which might lead to unexpected results